geode-core/src/test/java/org/apache/geode/internal/net/SocketCreatorJUnitTest.java

```diff
@@ -76,10 +80,27 @@ public void testConfigureServerPlainSocketDoesntSetSoTimeout() throws Exception
     verify(socket, never()).setSoTimeout(timeout);
   }
 
+  @Test
+  public void testBindExceptionMessageFormatting() throws Exception {
+    final SocketCreator socketCreator = new SocketCreator(mock(SSLConfig.class));
+    final Socket socket = mock(Socket.class);
+
+    ServerSocket serverSocket = null;
+    try {
+      serverSocket = socketCreator.createServerSocket(11234, 10, null);
```

Comment:
Ideally there would be tests for both conditions of the new ternary operator in `BindException()` in `createServerSocket`

Reply:
That's a good idea - I will add the missing test before merging.